From a41d41a75241a16d4ad2f10434efcbfc4d97f503 Mon Sep 17 00:00:00 2001
From: Eli Collins <eli@apache.org>
Date: Thu, 23 Aug 2012 20:49:55 +0000
Subject: [PATCH 0410/1357] HDFS-3715. Fix TestFileCreation#testFileCreationNamenodeRestart. Contributed by Andrew Wang

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/trunk@1376689 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 7a4bd856a92e1e66846acdc0838c963bdc5f772c)

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/trunk@1376698 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 124843a67c2c7fd377e952f7196b24a80a5b23d6)
---
 .../org/apache/hadoop/hdfs/TestFileCreation.java   |    6 +----
 .../hdfs/TestFileCreationNamenodeRestart.java      |   26 --------------------
 2 files changed, 1 insertions(+), 31 deletions(-)
 delete mode 100644 hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestFileCreationNamenodeRestart.java

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestFileCreation.java b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestFileCreation.java
index 13cb5ce..fa5ebab 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestFileCreation.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestFileCreation.java
@@ -80,7 +80,6 @@ import org.apache.hadoop.security.UserGroupInformation;
 import org.apache.hadoop.test.GenericTestUtils;
 import org.apache.hadoop.util.Time;
 import org.apache.log4j.Level;
-import org.junit.Ignore;
 import org.junit.Test;
 
 /**
@@ -586,12 +585,9 @@ public class TestFileCreation {
 
   /**
    * Test that file leases are persisted across namenode restarts.
-   * This test is currently not triggered because more HDFS work is 
-   * is needed to handle persistent leases.
    */
-  @Ignore
   @Test
-  public void xxxtestFileCreationNamenodeRestart() throws IOException {
+  public void testFileCreationNamenodeRestart() throws IOException {
     Configuration conf = new HdfsConfiguration();
     final int MAX_IDLE_TIME = 2000; // 2s
     conf.setInt("ipc.client.connection.maxidletime", MAX_IDLE_TIME);
diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestFileCreationNamenodeRestart.java b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestFileCreationNamenodeRestart.java
deleted file mode 100644
index bc86705..0000000
--- a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestFileCreationNamenodeRestart.java
+++ /dev/null
@@ -1,26 +0,0 @@
-/**
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.apache.hadoop.hdfs;
-import org.junit.Test;
-
-public class TestFileCreationNamenodeRestart {
-  @Test
-  public void testFileCreationNamenodeRestart() throws Exception {
-    new TestFileCreation().xxxtestFileCreationNamenodeRestart();
-  }
-}
-- 
1.7.0.4

